masterlist.yaml

```diff
@@ -907,6 +907,17 @@ plugins:
         util: '[TES5Edit v4.0.4b](https://www.nexusmods.com/skyrim/mods/25859)'
         itm: 36
 
+  - name: 'InigoMCM.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrim/mods/70035/'
+        name: 'InigoMCM'
+    req:
+      - *SKSE
+      - *SkyUI
```

Comment:
SkyUI requires SKSE, so you don't need to specify both as requirements.

```yaml
    req: [ *SkyUI ]
```

Reply:
While that may be true, stating both of them has been done for multiple plugins within the SLE masterlist, so I followed that as usual.

Reply:
Ah I see, the SSE masterlist doesn't feature the `*SkyUI` anchor, but rather it's done like so:
```yaml
  - name:  'StormLightning.esp'
    msg: [ *requiresMCM ]
    req: [ *SKSE ]
```
with
```yaml
  - &requiresMCM
    <<: *requiresMCM_X
    subs: [ '[SkyUI SE](https://www.nexusmods.com/skyrimspecialedition/mods/12604/)' ]
    condition: 'not active("SkyUI_SE.esp")'
```

Reply:
It used to have that anchor, but there are very few mods that explicitly require SkyUI to function, so it was removed.